config/routes.rb

```diff
@@ -8,11 +8,13 @@
   root "users#index"
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show, :new, :create, :destroy] do
-      resources :comments, only: [:new, :create, :destroy]
+      resources :comments, only: [:new, :create, :destroy, :index]
       resources :likes, only: [:new, :create]
```

Comment:
- You have done well with your project so far, but according to the project requirements, you are to configure new controllers specifically for your api-endpoints, since you are required to configure new controllers, please bear in mind you'd also need to set up new routes for your API-endpoints. Please fix this
- Please consider using namespaces for your new API endpoints so it can be easy to differentiate them from your existing routes. e.g the expected endpoint to get all comments belonging to a post should look like this:
   - ```ruby
      /users/:user_id/posts/:id/comments
      ```

Reply:
Fixed.